src/item.cpp

```diff
@@ -179,7 +144,7 @@ Item::Item(const rapidjson::Value &json) :
             if (!prop.HasMember("name") || !prop["name"].IsString() || !prop.HasMember("values") || !prop["values"].IsArray())
                 continue;
             std::string name = prop["name"].GetString();
-            if (name == "Map Level")
```

Comment:
Yes it should, it used to be map level in the past but they changed it

Reply:
I don't think that's going to end well, Map Tier would be e.g. 1-16, while Level is Item Level(ilvl JSON value, outsite of the properties array), for maps being ~68-84. That or required player level.

As the program seems to be working fine without the collapsing of map tier onto the (item) level field & filters (and their are 3 different search input fields for these concepts), I think this conditional small block should be removed.